extensions/Worker.Extensions.Http/src/DefaultFromBodyConversionFeature.cs

```diff
@@ -56,19 +56,19 @@ internal class DefaultFromBodyConversionFeature : IFromBodyConversionFeature
             }
             else if (targetType == typeof(byte[]))
             {
-                result = ReadBytes(requestData, context.CancellationToken);
+                result = ReadBytes(requestData);
             }
             else if (targetType == typeof(Memory<byte>))
             {
-                Memory<byte> bytes = ReadBytes(requestData, context.CancellationToken);
+                Memory<byte> bytes = ReadBytes(requestData);
                 result = bytes;
             }
             else if (HasJsonContentType(requestData))
             {
                 ObjectSerializer serializer = requestData.FunctionContext.InstanceServices.GetService<IOptions<WorkerOptions>>()?.Value?.Serializer
                  ?? throw new InvalidOperationException("A serializer is not configured for the worker.");
 
-                result = serializer.Deserialize(requestData.Body, targetType, context.CancellationToken);
+                result = serializer.Deserialize(requestData.Body, targetType, CancellationToken.None);
```

Comment:
Do you want to change to `DeserializeAsync` in this PR, or keep that change separate?

Reply:
I'd like to keep them separate PRs